### PR TITLE
Move image customizer docs to common docs location 

### PR DESCRIPTION
**Pull request by romoh**
### **Checklist**
- [ ] Tests added/updated
- [X] Documentation updated (if needed)
- [ ] Code conforms to style guidelines

### CODE REVIEW

File: toolkit/tools/imagecustomizer/docs/building-packages.md [hunk 1]
```diff
@@ -1,97 +0,0 @@
-# Building custom packages
```
Comment (cwize1) [inline toolkit/tools/imagecustomizer/docs/building-packages.md:1]: Why was this file deleted?

**Review by cwize1 — CHANGES_REQUESTED** [review]

File: toolkit/tools/imagecustomizer/docs/building-packages.md (hunk 1 above)
Comment (romoh) [reply toolkit/tools/imagecustomizer/docs/building-packages.md:1]: This is distro focused, it should live in azl repo IMO

Comment (cwize1) [reply toolkit/tools/imagecustomizer/docs/building-packages.md:1]: I created this doc to be a simplified way of building packages. The AZL repo’s instructions use the toolkit/toolchain, which heavily complicates things.

Please restore this file.

**Review by cwize1 — DISMISSED** [review]

**Review by cwize1 — APPROVED** [review]

**Review by gmileka — APPROVED** [review]

**Comment by romoh** [comment]
New PR https://github.com/microsoft/azure-linux-image-tools/pull/35
